### PR TITLE
Exchange listings

### PR DESCRIPTION
**Pull request by matt-auckland**
Adds Coinmerce, Vertbase, Altilly, Nova Exchange, Equicex and CoinXtr.

Brings the other language's Buy NavCoin up to date as well.

### CODE REVIEW

File: content/buy-navcoin/index.en.md [hunk 1]
```diff
@@ -127,6 +133,36 @@ newTab="true"
...
+          titleText="Altilly"
+          imgSrc="/images/buy-navcoin/altilly.jpg"
+          text="BTC / NAV"
+          linkUrl="https://www.altilly.com/asset/NAV"
+      >}}
+     {{< exchange
+          titleText="Nova Exchange"
+          imgSrc="/images/buy-navcoin/nova_exchange.jpg"
+          text="BTC + ETH / NAV"
+          linkUrl="https://bitqist.com/exchange/nav-coin"
```
Comment (marcus290) [inline content/buy-navcoin/index.en.md]: url should be Nova Exchange for all languages

Comment (matt-auckland) [reply content/buy-navcoin/index.en.md]: Good catch

Comment (matt-auckland) [inline content/buy-navcoin/index.en.md]:
```suggestion
          linkUrl="https://novaexchange.com/market/BTC_NAV/"
```

File: content/buy-navcoin/index.es.md [hunk 2]
```diff
@@ -37,84 +37,132 @@ newTab="true"
...
+          titleText="Altilly"
+          imgSrc="/images/buy-navcoin/altilly.jpg"
+          text="BTC / NAV"
+          linkUrl="https://www.altilly.com/asset/NAV"
+      >}}
+     {{< exchange
+          titleText="Nova Exchange"
+          imgSrc="/images/buy-navcoin/nova_exchange.jpg"
+          text="BTC + ETH / NAV"
+          linkUrl="https://bitqist.com/exchange/nav-coin"
```
Comment (matt-auckland) [inline content/buy-navcoin/index.es.md]:
```suggestion
          linkUrl="https://novaexchange.com/market/BTC_NAV/"
```

File: content/buy-navcoin/index.jp.md [hunk 3]
```diff
@@ -127,8 +133,39 @@ newTab="true"
...
+          titleText="Altilly"
+          imgSrc="/images/buy-navcoin/altilly.jpg"
+          text="BTC / NAV"
+          linkUrl="https://www.altilly.com/asset/NAV"
+      >}}
+     {{< exchange
+          titleText="Nova Exchange"
+          imgSrc="/images/buy-navcoin/nova_exchange.jpg"
+          text="BTC + ETH / NAV"
+          linkUrl="https://bitqist.com/exchange/nav-coin"
```
Comment (matt-auckland) [inline content/buy-navcoin/index.jp.md]:
```suggestion
          linkUrl="https://novaexchange.com/market/BTC_NAV/"
```

File: content/buy-navcoin/index.kr.md [hunk 4]
```diff
@@ -127,8 +127,45 @@ newTab="true"
...
+          titleText="Altilly"
+          imgSrc="/images/buy-navcoin/altilly.jpg"
+          text="BTC / NAV"
+          linkUrl="https://www.altilly.com/asset/NAV"
+      >}}
+     {{< exchange
+          titleText="Nova Exchange"
+          imgSrc="/images/buy-navcoin/nova_exchange.jpg"
+          text="BTC + ETH / NAV"
+          linkUrl="https://bitqist.com/exchange/nav-coin"
```
Comment (matt-auckland) [inline content/buy-navcoin/index.kr.md]:
```suggestion
          linkUrl="https://novaexchange.com/market/BTC_NAV/"
```

File: content/buy-navcoin/index.ru.md [hunk 5]
```diff
@@ -127,6 +127,42 @@ newTab="подлинный"
...
+          titleText="Altilly"
+          imgSrc="/images/buy-navcoin/altilly.jpg"
+          text="BTC / NAV"
+          linkUrl="https://www.altilly.com/asset/NAV"
+      >}}
+     {{< exchange
+          titleText="Nova Exchange"
+          imgSrc="/images/buy-navcoin/nova_exchange.jpg"
+          text="BTC + ETH / NAV"
+          linkUrl="https://bitqist.com/exchange/nav-coin"
```
Comment (matt-auckland) [inline content/buy-navcoin/index.ru.md]:
```suggestion
          linkUrl="https://novaexchange.com/market/BTC_NAV/"
```

File: content/buy-navcoin/index.zh.md [hunk 6]
```diff
@@ -127,6 +127,42 @@ newTab="正确的"
...
+          titleText="Altilly"
+          imgSrc="/images/buy-navcoin/altilly.jpg"
+          text="BTC / NAV"
+          linkUrl="https://www.altilly.com/asset/NAV"
+      >}}
+     {{< exchange
+          titleText="Nova Exchange"
+          imgSrc="/images/buy-navcoin/nova_exchange.jpg"
+          text="BTC + ETH / NAV"
+          linkUrl="https://bitqist.com/exchange/nav-coin"
```
Comment (matt-auckland) [inline content/buy-navcoin/index.zh.md]:
```suggestion
          linkUrl="https://novaexchange.com/market/BTC_NAV/"
```

File: content/buy-navcoin/index.en.md (hunk 1 above)
Comment (matt-auckland) [reply content/buy-navcoin/index.en.md]: Fixed

**Pull request merged**